### PR TITLE
chore: Update PR template

### PR DESCRIPTION
**Pull request by tihuan**
Update PR template

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## Checklist 🛎️
+
+- [ ] Add product, design, and eng for rdev review
+
+- [ ] For UI changes, add screenshots/videos, so the reviewers know what you expect them to see
+
+- [ ] For UI changes, add e2e tests to prevent regressions
+
```
Comment (atolopko-czi) [inline PULL_REQUEST_TEMPLATE.md:8]: Do we want this to appear _before_ the issue link? Would prefer issue at top.

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## Checklist 🛎️
+
+- [ ] Add product, design, and eng for rdev review
```
Comment (atolopko-czi) [inline PULL_REQUEST_TEMPLATE.md:3]: Do this mean as PR reviewers?

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+
+- [ ] For UI changes, add screenshots/videos, so the reviewers know what you expect them to see
+
+- [ ] For UI changes, add e2e tests to prevent regressions
```
Comment (atolopko-czi) [inline PULL_REQUEST_TEMPLATE.md:7]: 💯

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (tihuan) [reply PULL_REQUEST_TEMPLATE.md]: Yes! I updated the sentence to: `Add product, design, and eng as reviewers for rdev review` does that look better?

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (tihuan) [reply PULL_REQUEST_TEMPLATE.md]: Ooh good point - moved to the bottom!

**Comment by tihuan** [comment]
@atolopko-czi just addressed your great feedback! PTAL again thank you 😄 🙏

**Review by atolopko-czi — APPROVED** [review]

**Pull request merged**